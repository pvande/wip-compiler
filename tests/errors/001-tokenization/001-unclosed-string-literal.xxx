a := 1
x : string = "This is an unclosed string
b := 2
