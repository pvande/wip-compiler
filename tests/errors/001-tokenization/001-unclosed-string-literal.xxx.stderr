Error: "String literal is unterminated"
In [1;37mtests/errors/001-tokenization/001-unclosed-string-literal.xxx[0m on line [1;37m2[0m

> [0;36mx : string = "This is an unclosed string[0m
  [0;31m             ^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
